en/assembly/quick_start_cuav_v5_plus.md

```diff
@@ -66,11 +67,13 @@ The figure below shows how you can access your remote receiver (please find the
 
 ## Spektrum Satellite Receivers
 
-The V5+ has a dedicated DSM cable. The Spektrum satellite receiver should be connected to the flight control DSM/SBUS/RSSI interface.
+The V5+ has a dedicated DSM cable.
+The Spektrum satellite receiver should be connected to the flight control DSM/SBUS/RSSI interface.
 
 ## Power
 
-The V5+ kit includes HV\_PM, which supports 2~10s LiPo battery. Please connect the 6pin connector of the HW\_PM module to the flight control power1 interface.
+The V5+ kit includes the HV\_PM, which supports 2~10s LiPo battery.
```

Comment:
@cuhome - As above, would be good to outline how to power the servo rail. Also a note on how to use Power2.

Reply:
I will describe how to modify it to make it similar to power1 with adc detection.

Reply:
Need independent bec to power the servo rail